media-tv/plex-media-server/files/etc/revdep-rebuild/80plexmediaserver

```diff
@@ -0,0 +1 @@
+SEARCH_DIRS_MASK="/usr/lib64/plexmediaserver/lib /usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload /usr/lib/plexmediaserver/Resources/Python/lib/python2.7/site-packages/lxml"
```

Comment:
On x86, do we need /usr/lib/plexmediaserver/lib?

Reply:
yes, however there is no /usr/lib64/plexmediaserver, so I need to fix that..

Reply:
This has been updated